### ISSUE TITLE
Not entirely clear what you wish to plot

### ISSUE THREAD

**Issue by ZebNemeth** [frantisek901/mandelbrot#1]
You mention wanting to plot, in 3D;
x-axis : Re(c)
y-axis ; Im(c)
z-axis : exponent k

Since z_0 = c, I derived that you mean;
A semi-connected stack of differently-exponentiated mandelbrot sets.

Is that correct?
If so, Paul Nylander has some Mathematica code shared: https://nylander.wordpress.com/2009/07/06/3d-exponential-mandelbrot-set/

**Comment by frantisek901** [comment]
Exactly! Stack of differently exponentiated Mandelbrot sets!Dne 31. 1. 2021 1:32 napsal uživatel Zeb Németh <notifications@github.com>:
You mention wanting to plot, in 3D;
x-axis : Re(c)
y-axis ; Im(c)
z-axis : exponent k
...so really, a semi-connected stack of differently-exponentiated mandelbrot sets? Or something else?

**Comment by ZebNemeth** [comment]
Then you and I think alike. I'd like to think along, but am not well-versed in R, but have some clear ideas on how to do so in Python.

Good luck, and if you want to spar on how to tackle the problem, we can maybe do so on Discord.
